perfmetrics/scripts/ml_tests/tf/resnet/setup_host_and_run_model.sh

```diff
@@ -20,7 +20,7 @@ BUCKET_TYPE=$1
 cd "$HOME/github/gcsfuse/perfmetrics/scripts"
 
 echo "Setting up the machine with Docker and Nvidia Driver..."
-DRIVER_VERSION="520.61.05"
```

Comment:
Do we require to change driver version [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/ml_tests/pytorch/v2/dino/setup_host_and_run_container.sh#L22) as well.